Packs/SecureHealth/Scripts/SecureHealthFIrewallAPI/SecureHealthFIrewallAPI.yml

```diff
@@ -0,0 +1,31 @@
+args:
```

Comment:
this script should be converted to an integration, each action is a different command